src/endpoints/http/collection.js

```diff
@@ -0,0 +1,25 @@
+function message(message) {
+  return { message: message };
+}
+
+/**
+ * Add the route to create a new collection for the users App.
+ * This route expects a db connection to be present on the request object.
+ *
+ * @param {object} server - The express server.
+ */
+export function createCollection(server) {
+  server.post("/collection", (req, res) => {
+    if (!req.body.name) {
+      return res.status(400).send(message('name is required'));
+    }
+
+    req.db.createCollection(req.body.name, err => {
+      if (err) {
+        return res.status(500).send(message(err.message));
```

Comment:
might be better to just call `next(err)`. This way the global error handler will be called and it will log the error as well. Also we can make sure the errors are in the same format